src/test/java/build/buf/protovalidate/FormatTest.java

```diff
@@ -15,107 +15,210 @@
-
-    assertThatThrownBy(
-            () -> {
-              Format.format("first value: %s and  %s", val);
-            })
-        .isInstanceOf(ErrException.class)
-        .hasMessageContaining("format: not enough arguments");
+  // Version of the cel-spec that this implementation is conformant with
+  // This should be kept in sync with the version in gradle.properties
+  private static String CEL_SPEC_VERSION = "v0.24.0";
+
+  private static Env env;
+
+  private static List<SimpleTest> formatTests;
+  private static List<SimpleTest> formatErrorTests;
+
+  private static List<String> SKIPPED_TESTS =
+      Arrays.asList(
+          // Success Tests
+          // found no matching overload for 'format' applied to 'string.(list(type(string)))'
+          "type() support for string",
+          // found no matching overload for 'format' applied to 'string.(list(map(string, dyn)))'
+          "map support for string",
+          // found no matching overload for 'format' applied to 'string.(list(map(dyn, dyn)))'
+          "map support (all key types)",
+          // found no matching overload for 'format' applied to 'string.(list(map(dyn, dyn)))'
+          "dyntype support for maps",
+          // Error Tests
+          // found no matching overload for 'format' applied to
+          // 'string.(list(cel.expr.conformance.proto3.TestAllTypes))'
+          "object not allowed",
+          // found no matching overload for 'format' applied to 'string.(list(map(int, dyn)))'
+          "object inside map");
+
+  @BeforeAll
+  static void setUp() {
+    try {
+      byte[] encoded =
+          Files.readAllBytes(
+              Paths.get(
+                  "src/test/resources/testdata/string_ext_" + CEL_SPEC_VERSION + ".textproto"));
+      String data = new String(encoded, StandardCharsets.UTF_8);
+      SimpleTestFile.Builder bldr = SimpleTestFile.newBuilder();
+      TextFormat.getParser().merge(data, bldr);
+      SimpleTestFile testData = bldr.build();
+
+      List<SimpleTestSection> sections = testData.getSectionList();
+
+      // Find the format tests which test successful formatting
+      // Defaults to an empty list if nothing is found
+      formatTests =
+          sections.stream()
+              .filter(s -> s.getName().equals("format"))
+              .findFirst()
+              .map(SimpleTestSection::getTestList)
+              .orElse(Collections.emptyList());
+
+      // Find the format error tests which test errors during formatting
+      // Defaults to an empty list if nothing is found
+      formatErrorTests =
+          sections.stream()
+              .filter(s -> s.getName().equals("format_errors"))
+              .findFirst()
+              .map(SimpleTestSection::getTestList)
+              .orElse(Collections.emptyList());
+
+      env = Env.newEnv(Library.Lib(new ValidateLibrary()));
+
+    } catch (InvalidPathException | IOException e) {
+      fail(e);
```

Comment:
I'd just add a `throws Exception` to the `setUp` method which will automatically fail the test.